oak-jackrabbit-api/src/main/java/org/apache/jackrabbit/api/JackrabbitNode.java

```diff
@@ -103,4 +103,8 @@ void setMixins(String[] mixinNames)
             return null;
         }
     }
+
+    default long getEstimatedChildNodes() throws RepositoryException {
```

Comment:
the method name is a bit confusing ...
```suggestion
    default long getEstimatedChildNodeCount() throws RepositoryException {
```